### PR TITLE
Feature/prompt list 3

### PR DESCRIPTION
**Pull request by RiFactor**
Getting prompts from the server

### CODE REVIEW

**Review by JR-Mitchell — APPROVED** [review]
LGTM

File: client/src/pages/MainPage.tsx [hunk 1]
```diff
@@ -89,6 +91,39 @@ const MainPage = () => {
     };
   }, [ws, allowList, gameState]);
 
+  // TODO is this needed?
```
Comment (JR-Mitchell) [inline client/src/pages/MainPage.tsx:94]: You can re-delete this if you want

**Review by JR-Mitchell — APPROVED** [review]
LGTM again

**Review by fjebaker — APPROVED** [review]
:shipit:

**Pull request merged**